### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by mohametalmeari**
- [ ] Create a table named owners
- [ ] Create a table named species
- [ ] Modify animals table
- [ ] Insert data into the owners and species tables
- [ ] Write queries (using JOIN)
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -53,3 +53,73 @@ FROM
...
+    species.name;
+
+SELECT
+    owners.full_name,
+    animals.name
+FROM
+    animals
+INNER JOIN owners
+    ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Jennifer Orwell';
```
Comment (muneebulrehman) [inline queries.sql:103]: As per the project requirements, this query should fetch only the `Digimons` that belong to Jennifer Orwell. However, your query fetches all the animals she owns. Kindly fix this error.

File: queries.sql [hunk 2]
```diff
@@ -53,3 +53,73 @@ FROM
...
+
+SELECT
+    owners.full_name,
+    animals.name
+FROM
+    animals
+INNER JOIN owners
+    ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts IS NOT NULL;
```
Comment (muneebulrehman) [inline queries.sql:112]: As per the project requirements, this query should return all the animals owned by Dean Winchester that **haven't tried to escape**. This query doesn't return the right values. You are testing for `IS NOT NULL` for the `escape_attempts`. However, we need to test for value 0. Both the animals owned by Dean Winchester have tried to escape. So you query should return empty. However, it returns both the animals as per your screenshot. Kindly fix this error.

File: schema.sql [hunk 3]
```diff
@@ -9,3 +9,33 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals ADD COLUMN species VARCHAR(100);
+
```
Comment (muneebulrehman) [inline schema.sql:12]: As per the project requirements, you also need to remove the column `species` from the animals now that you have created a separate table `species`. Kindy do so.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @mohametalmeari , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job adding the correct data.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @mohametalmeari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mohametalmeari** [comment]
Thank you 🙏

**Pull request merged**